bindings/nodejs/lib/types/block/input/input.ts

```diff
@@ -62,6 +63,28 @@ class UTXOInput extends Input {
         this.transactionId = transactionId;
         this.transactionOutputIndex = transactionOutputIndex;
     }
+
+    /**
+     * Creates a `UTXOInput` from an output id.
+     */
+    static fromOutputId(outputId: OutputId): UTXOInput {
```

Comment:
Maybe we can implement this in Rust and expose it? I found this easier, although things like `TRANSACTION_ID_LENGTH` are not used in node so far.

Reply:
Yes, this should be implemented in rust

Reply:
So we need to Create a `UTXOInput` in Rust and convert it to back JS? or create a utility function? I'm not sure how that can be done in an idiomatic way.

Reply:
That would have to go through the message interface